pkg/bootkube/create.go

```diff
@@ -44,6 +46,9 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, createFn); err != nil {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
+
+	fmt.Println("Spawning self-hosted control plane...")
```

Comment:
It seems like the self-hosted control plane is in the process of being launched as soon as the assets are submitted to the bootstrap control plane which is when this function successfully returns.

The other place to put it would be inside the beginning of `WaitUntilPodsRunning` but then it will actually print earlier since these functions run concurrently. So the end of this function seemed the most accurate. Also putting it in `WaitUntilPodsRunning` will make it racey against the message `Sending self-hosted assets to temporary control plane...`

Its technically racey either way but in practice the way things are now, the messages should always be in the same order.